demos/hide_keyboard_bot/demo.py

```diff
@@ -0,0 +1,75 @@
+"""The module is a script to run the bot."""
+
+from hammett.core import Button, Screen
+from hammett.core.application import Application
+from hammett.core.constants import DEFAULT_STATE, SourcesTypes
+from hammett.core.mixins import StartMixin
+from hammett.core.persistence import RedisPersistence
+
+NEXT_SCREEN_DESCRIPTION = (
+    "It's magic! The previous screen has lost the keyboard! The same is going "
+    "to happen with this screen as well. Try it out."
+)
+
+START_SCREEN_DESCRIPTION = (
+    'Welcome to HammettHideKeyboardBot!\n'
+    '\n'
+    'This message will drop the keyboard after jumping to the next screen.'
+)
+
+
+class BaseScreen(Screen):
+    """The base screen for all screens in the bot."""
+
+    hide_keyboard = True
+
+
+class NextScreen(BaseScreen):
+    """The class implements the NextScreen screen."""
+
+    description = NEXT_SCREEN_DESCRIPTION
+
+    async def add_default_keyboard(self, _update, _context):
+        """Set up the default keyboard for the screen."""
+        return [[
+            Button(
+                '⬅️ Back',
+                StartScreen,
+                source_type=SourcesTypes.JUMP_SOURCE_TYPE,
+            ),
+        ]]
+
+
+class StartScreen(BaseScreen, StartMixin):
+    """The class implements the StartScreen screen. It is a response
+    to the /start command.
```

Comment:
`The class implements StartScreen, which acts as a response to the /start command.`